test/images/agnhost/Dockerfile_windows

```diff
@@ -26,8 +26,7 @@ ADD https://download.savannah.nongnu.org/releases/coreutils/windows-64bit-unsupp
 # we're also creating an empty /uploads folder, which we're copying over to Windows  because
 # we can't RUN commands on a Windows image.
 RUN tar -xzvf /coredns.tgz &&\
-    unzip iperf.zip &&\
-    mv iperf-2.0.9-win64 iperf &&\
+    unzip -j -d /bin iperf.zip &&\
```

Comment:
Keep in mind that this will unzip iperf.zip into the Linux Container's /bin folder, which won't really solve the issue. In fact, this will cause the image build to fail.

Instead, I think we can change the line 48 from:

```
COPY --from=prep /iperf /iperf
```

to:

```
COPY --from=prep /iperf/* /bin
```

From what I've checked, thata seems to work, the image builds succesfully, and iperf cand be used directly. I've built the image docker.io/claudiubelu/agnhost:2.35 if you need to check.